packages/salesforcedx-utils/test/jest/cli/commandOutput.test.ts

```diff
@@ -93,16 +97,45 @@ describe('CommandOutput Unit Tests.', () => {
     });
   });
 
-  it('Should have data from stdout on failure if there is no stderr.', async () => {
+  it('Should have data from stdout on failure if there is no stderr when command not sf.', async () => {
+    fakeExecution.command.command = 'notsf';
     const stdoutCallback =
       fakeExecution.stdoutSubject.subscribe.mock.calls[0][0];
     stdoutCallback(goodOutput);
     const exitCallback =
       fakeExecution.processExitSubject.subscribe.mock.calls[0][0];
-    // Call the exit callback with a 0 response to indicate success
+    // Call the exit callback with a 1 response to indicate failure
```

Comment:
Good catch! 😅

Should this comment change be made to all failure cases?  (Lines 93 and 122 still say the success message.)

Reply:
Will do